src/tests/work-item/work-item-list/work-item-list.spec.js

```diff
@@ -65,7 +65,7 @@ describe('Work item list', function () {
       expect(page.workItemDescription(page.workItemByNumber(0))).toBe('Some Description');
       expect(page.workItemTitle(page.workItemByNumber(0))).toBe('Some Title');
       page.allWorkItems.getText().then(function (text) {
-        expect(text).toContain("View Details Delete\nnew\n1\nSome Title\nSome Description");
+        expect(text).toContain("new\n1\nSome Title 1\nSome Description 1\nView Details Delete");
```

Comment:
Ah, and that was was happening here effectively and was the cause of the error. In the old code, the buttons were defined before the content but displayed after the content. I "ordered" that so that the hierarchy matches the display and it broke. So it not checks the display but the hierarchy, what I think is not the point. I'll talk with Len about that.